github/ci/prow-deploy/files/jobs/kubevirt/hostpath-provisioner/hostpath-provisioner-presubmits.yaml

```diff
@@ -33,3 +33,65 @@ presubmits:
           resources:
             requests:
               memory: "29Gi"
+  - name: pull-hostpath-provisioner-sanity
+    skip_branches:
+      - release-\d+\.\d+
+    annotations:
+      fork-per-release: "true"
+    always_run: false
+    optional: true
+    decorate: true
+    decoration_config:
+      timeout: 1h
+      grace_period: 5m
+    max_concurrency: 6
+    labels:
+      preset-dind-enabled: "true"
```

Comment:
Do we need dind for the sanity tests?

Reply:
Yes, so the sanity check creates a docker image to run the test in. Part of the test will create a UDS to test various aspects of the CSI plugin.